README.md

```diff
@@ -16,6 +16,25 @@ Use cases may include:
 - Github (v3 API)
 
 
+<h2>Configuration File</h2>
+Freighter requires the presence of a .freighter file with any of the following supported extensions:
```

Comment:
...and by optional, i mean a strategy, i.e look for config file then fallback to command line args, then fall back to env variable.